packages/components/psammead-story-promo/src/IndexAlsos/index.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { Fragment } from 'react';
+import styled from 'styled-components';
+import { node, string, shape } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import { C_EBON, C_METAL, C_LUNAR } from '@bbc/psammead-styles/colours';
+import { getBrevier } from '@bbc/gel-foundations/typography';
+import { getSerifMedium } from '@bbc/psammead-styles/font-styles';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+
+const StyledIndexAlsos = styled.div`
+  position: relative;
+  z-index: 2;
+  padding: ${GEL_SPACING_DBL} 0 ${GEL_SPACING};
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    display: none;
+    visibility: hidden;
+  }
+`;
+
+const StyledIndexAlso = styled.div`
+  border-top: 1px solid ${C_LUNAR};
+  padding: ${GEL_SPACING} 0;
+`;
+
+const StyledIndexAlsosUl = styled.ul`
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+`;
+
+const IndexAlsosMediaIndicator = styled.span`
+  padding-right: ${GEL_SPACING};
+`;
+
+const IndexAlsosText = styled.span`
+  display: inline;
+  vertical-align: middle;
+`;
+
+const StyledIndexAlsosLink = styled.a`
+  ${props => (props.script ? getBrevier(props.script) : '')};
+  ${({ service }) => getSerifMedium(service)}
+  color: ${C_EBON};
+  text-decoration: none;
+
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+  }
+
+  &:visited {
+    color: ${C_METAL};
+  }
+`;
+
+const IndexAlsosLink = ({
+  children,
+  script,
+  service,
+  url,
+  mediaIndicator,
+  mediaType,
+}) => {
+  return (
+    <StyledIndexAlsosLink href={url} script={script} service={service}>
+      {mediaIndicator ? (
+        <Fragment>
+          <IndexAlsosMediaIndicator>{mediaIndicator}</IndexAlsosMediaIndicator>
+          <span // eslint-disable-next-line jsx-a11y/aria-role
+            role="text"
```

Comment:
This is just me being picky, but this could be replaced with a Styled Components `span`:

```
const SpanAttr = styled.span.attrs({
  role: 'text',
})``;
```

I just find the `eslint-disable` in the middle of the tag a bit icky, but that's just my preference

Reply:
Done 🕺🏻